benchmarks/timm_models.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+import importlib
+import logging
+import os
+import re
+import subprocess
+import sys
+import warnings
+
+import torch
+from common import BenchmarkRunner
+from common import main
+
+import torchdynamo
+from torchdynamo.testing import collect_results
+from torchdynamo.utils import clone_inputs
+
+
+def pip_install(package):
+    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
+
+
+try:
+    importlib.import_module("timm")
+except ModuleNotFoundError:
+    print("Installing Pytorch Image Models...")
+    pip_install("git+https://github.com/rwightman/pytorch-image-models")
+finally:
+    from timm.data import resolve_data_config
+    from timm.models import create_model
+
+SKIP = set()
+
+TIMM_MODELS = dict()
+filename = "timm_models_list.txt"
+if os.path.exists("benchmarks"):
+    filename = "benchmarks/" + filename
+with open(filename, "r") as fh:
+    lines = fh.readlines()
+    lines = [line.rstrip() for line in lines]
+    for line in lines:
+        model_name, batch_size = line.split(" ")
+        TIMM_MODELS[model_name] = int(batch_size)
+
+
+# TODO - Figure out the reason of cold start memory spike
```

Comment:
@ngimel These are the models that pass eager, but fail when we go through the Dynamo-AOT-Nfuser pipeline. We have to understand why it fails.